orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -198,21 +198,21 @@
              {% if 'ATLAS' in exposed and 'ATLAS_SERVER' in salt['pillar.get']('gateway:location') -%}
              <param>
                  <name>ATLAS</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
              </param>
              <param>
                  <name>ATLAS-API</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
```

Comment:
This would break Atlas, we should not be using Knox HA for atlas since it is active-passive. adding stickysession would break it badly.

Reply:
Ok so no changes to Atlas then?